test_package/.analysis_options

```diff
@@ -0,0 +1 @@
+{ analyzer: { exclude: ['**'] } }
```

Comment:
+1 for the YAML format. It's easier to read.

If there's a reason to use the JSON version, that's fine, too.